openhcl/openhcl_boot/src/main.rs

```diff
@@ -676,9 +686,12 @@ fn shim_main(shim_params_raw_offset: isize) -> ! {
 
     // Compute the ending boot time. This has to be before writing to device
     // tree, so this is as late as we can do it.
+
+    let boot_endtime = get_ref_time(p.isolation_type).unwrap_or(0);
+
     let boot_times = boot_reftime.map(|start| BootTimes {
         start,
-        end: minimal_rt::reftime::reference_time(),
+        end: boot_endtime,
```

Comment:
can we not inline the call above by doing `end: get_ref_time(...)` because of the variable capture?